examples/simple_tabset/app.R

```diff
@@ -0,0 +1,23 @@
+library(shiny)
+library(shiny.semantic)
+library(highlighter)
+library(magrittr)
+library(plotly)
```

Comment:
Do we need all these packages? `highlighter`?   `magrittr `?

Reply:
done